conda/fetch.py

```diff
@@ -25,9 +25,13 @@
 from .exceptions import (ProxyError, ChannelNotAllowed, CondaRuntimeError, CondaSignatureError,
                          CondaHTTPError)
 from .install import (add_cached_package, find_new_location, package_cache, dist2pair,
-                      rm_rf, exp_backoff_fn)
+                      rm_rf, exp_backoff_fn, on_win)
 from .lock import Locked as Locked
 from .utils import memoized
+from .exceptions import ProxyError, ChannelNotAllowed, CondaRuntimeError, \
+    CondaSignatureError, CondaIOError
```

Comment:
while it's just style, here I strongly prefer

    from .exceptions import (ProxyError, ChannelNotAllowed, CondaRuntimeError,
                             CondaSignatureError, CondaIOError)